packages/logger/src/index.ts

```diff
@@ -94,53 +95,35 @@ export class ConsoleLogger implements Logger {
     return this.level;
   }
 
-  /**
```

Comment:
We can forego the docs here because docs generation is smart enough to pull in, as a fallback, the description of this method from the `Logger` interface this class implements!

<img width="837" alt="Screenshot 2024-10-03 at 3 50 52 PM" src="https://github.com/user-attachments/assets/c3809827-21a0-41e1-b43f-cc0c35d29edb">